js/signin.js

```diff
@@ -0,0 +1,64 @@
+const emailErrorMessage = document.querySelector(".email-error-message");
+const emailInput = document.querySelector(".email-input");
+
+emailInput.addEventListener("blur", function () {
+  const EMAIL_REGEX = /[a-z0-9]+@[a-z]+\.[a-z]{2,3}/
+
+  if (emailInput.value === "") {
+    emailErrorMessage.textContent = "이메일을 입력해 주세요.";
+    emailInput.classList.add("error-border");
+    return;
+  }
+
+  if (emailInput.value.match(EMAIL_REGEX) === null) {
+    emailErrorMessage.textContent = "올바른 이메일 주소가 아닙니다.";
+    emailInput.classList.add("error-border");
+    return;
+  }
+
+  emailErrorMessage.textContent = "";
+  emailInput.classList.remove("error-border");
+});
+
+const passwordErrorMessage = document.querySelector(".password-error-message");
+const passwordInput = document.querySelector(".password-input");
+
+passwordInput.addEventListener("blur", function () {
+  if (passwordInput.value === "") {
+    passwordErrorMessage.textContent = "비밀번호를 입력해 주세요.";
+    passwordInput.classList.add("error-border");
+    return;
+  }
+
+  passwordErrorMessage.textContent = "";
+  passwordInput.classList.remove("error-border");
+});
+
+
+const eyeIconChange = document.querySelector(".eye-icon-change");
+
+eyeIconChange.addEventListener("click", function () {
+  if (passwordInput.type === "password"){
+    eyeIconChange.src = "./image/eye-on.png";
+    passwordInput.type = "text";
+    return;
+  }
+  eyeIconChange.src = "./image/eye-off.png";
```

Comment:
사소하지만 개행이 들어가면 좋을 것 같아요!
```suggestion
  }
  
  eyeIconChange.src = "./image/eye-off.png";
```